showcase/src/assets/translations/nl.json

```diff
@@ -109,6 +109,28 @@
         "SIMPLE_HORIZONTAL_SLIDER": "Eenvoudige horizontale schuifregelaar",
         "SIMPLE_VERTICAL_SLIDER": "Eenvoudige verticale schuifregelaar",
         "VERTICAL_RANGE_SLIDER": "Verticale schuifregelaar met bereik"
+      },
+      "TYPOGRAPHY": {
```

Comment:
Could you order these translations in alphabetical order so they are easier to locate?